runagent/utils/enums/framework.py

```
@@ -0,0 +1,157 @@
+        """Get the category of this framework"""
+        if self.is_default():
+            return "default"
+        elif self.is_pythonic():
+            return "pythonic"
+        elif self.is_webhook():
+            return "webhook"
+        else:
+            return "unknown"
+    # Class methods for string validation and conversion
+    @classmethod
+    def from_string(cls, framework_str: str) -> 'Framework':
```

Ruff (0.13.1)

Redefinition of unused `from_string` from line 47

(F811)